recirq/characterization/xeb/grid_parallel_two_qubit_xeb_test.py

```diff
@@ -0,0 +1,118 @@
+import os
+import numpy as np
+import pytest
+
+import cirq
+from recirq.characterization.xeb.grid_parallel_two_qubit_xeb import GridParallelXEBMetadata, \
+    LAYER_A, LAYER_B, collect_grid_parallel_two_qubit_xeb_data, \
+    compute_grid_parallel_two_qubit_xeb_results
```

Comment:
Generally in Cirq we use parentheses for line continuations.

Reply:
changed it